docs/03-tutorials/00-serverless/svls-02-create-git-function.md

```diff
@@ -84,8 +84,7 @@ Follow these steps:
         secretName: # "git-creds-basic" or "git-creds-key"
     ```
    
-    >**NOTE:** It is recommended to use small repositories to source your git functions. Using large multi-repos can cause problems with function controller stability as it may run out of CPU and i/o resources.
-
+    >**NOTE:** To avoid performance degradation caused by large git repositories and large monorepos, the function controller implements a configurable backoff period for source checkout based on `APP_FUNCTION_REQUEUE_DURATION`. This behavior can be disabled, allowing the controller to perform source checkout with every reconciliation loop by labeling the function CR with the label `serverless.kyma-project.io/continuousGitCheckout: true`
```

Comment:
```suggestion
    >**NOTE:** To avoid performance degradation caused by large Git repositories and large monorepos, Function Controller implements a configurable backoff period for the source checkout based on `APP_FUNCTION_REQUEUE_DURATION`. This behavior can be disabled, allowing the controller to perform the source checkout with every reconciliation loop by labeling the Function CR with the label `serverless.kyma-project.io/continuousGitCheckout: true`
```